server/src/resolvers/default.ts

```diff
@@ -1,6 +0,0 @@
-import { defaultFieldResolver } from 'graphql';
-import { wrapWithSynthetics } from './helpers';
-
-const defaultResolver = wrapWithSynthetics(defaultFieldResolver);
```

Comment:
Would love to maintain this behavior as well if we can to ensure we can artificially slow down any field in the schema.